cpp/ps0000/ps0000soln.cpp

```diff
@@ -1,8 +1,9 @@
-#include <iostream>
+#include <iostream> // 2/2

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`